### ISSUE TITLE
Issue with local run

### ISSUE THREAD

**Issue by Bipinjot** [AmruthPillai/Reactive-Resume#524]
Hi!

I cloned the code. I executed the command "npm run build" for the production build.
Following this, I executed these, to run the app locally:

npm install -g serve
serve -s build.

Result:
Serving!      
Local:  http://localhost:5000 
![image](https://user-images.githubusercontent.com/25786623/123623974-ac3ffc80-d82b-11eb-9a04-02a99bc75eb3.png)

But I cannot access it using the same url. What could be the issue?
![Screenshot (35)](https://user-images.githubusercontent.com/25786623/123623762-7733aa00-d82b-11eb-87b6-bccab45e13ed.png)

**Comment by coreysnodgrass** [comment]
Hi @Bipinjot - This project uses gatsbyjs (https://www.gatsbyjs.com/), your serve command may be able to serve the files in your folder, but I'm pretty sure you need gatsby to serve the files.

Setup your .env file with Google firebase values (create a new project) https://firebase.google.com/

use

1. `npm install`
2. `npm start` (for development)
3. `npm serve` (for production)

See - https://github.com/AmruthPillai/Reactive-Resume#readme and look at the steps.

**Comment by Bipinjot** [comment]
Hi @coreysnodgrass 

Thank you. It did help, I was able to access the URL on my local machine. However, it new error popped up. It's related to the API key.
I tried to use both firestore DB and the real-time DB to resolve this. But the issue persists on my machine. 
I also tried it on another machine, but it worked okay. I'm not sure what is the cause.
![image](https://user-images.githubusercontent.com/25786623/123790795-5edf9000-d8fc-11eb-92fc-7bde2fd85ed0.png)
